disk-utils/fdisk-menu.c

```diff
@@ -221,6 +221,7 @@ static const struct menu menu_dos = {
 		MENU_ENT('c', N_("toggle the dos compatibility flag")),
 
 		MENU_XENT('b', N_("move beginning of data in a partition")),
+		MENU_XENT('F', N_("fix partitions C/H/S values")),
```

Comment:
'F' is already used for "list free unpartitioned space" (menu_generic).

Reply:
Some other letters are also different in more menus...

`f` is used for fixing partition order, so chose `F` to fix C/H/S.

Reply:
Ah, you have it in menu_dos[] and in expert mode, then 'F" is fine. Sorry.